### PR TITLE
Add pre-commit hooks configuration and docs

### PR DESCRIPTION
**Pull request by pmac**
See https://pre-commit.com/ for details

## Testing

You can try it by checking out this branch, installing pre-commit using instructions on their site (I used brew for Mac), and then running `pre-commit install`. You can try running it against all of the files by running `pre-commit run --all-files`.

### CODE REVIEW

**Review by Aayushap1 — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            git-repos|
+            lib/fluent_migrations
+          )
+  - repo: https://github.com/awebdeveloper/pre-commit-stylelint
+    rev: 0.0.2
+    hooks:
+      - id: stylelint
+        additional_dependencies:
+          - "stylelint@13.2.1"
+          - "stylelint-config-standard@20.0.0"
```
Comment (alexgibson) [inline .pre-commit-config.yaml:24]: I think this can be updated to `13.12.0`

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v6.3.0
+    hooks:
+    -   id: eslint
+        additional_dependencies:
+          - "eslint@6.3.0"
```
Comment (alexgibson) [inline .pre-commit-config.yaml:30]: Update to `7.0.0`

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          - "eslint-plugin-no-jquery@2.5.0"
```
Comment (alexgibson) [inline .pre-commit-config.yaml:31]: This can now be removed

File: setup.cfg [hunk 4]
```diff
@@ -2,20 +2,15 @@
...
-    bedrock/externalfiles/files_cache
-    mofo_security_advisories
-    release_notes
-    content_cards
-    product_details_json
-    community_data
-    www_config
+    .git,
+    __pycache__,
+    docs,
```
Comment (alexgibson) [inline setup.cfg:7]: should we add `assets` here now as well?

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        exclude: >
+          (?x)^(
+            docs|
```
Comment (alexgibson) [inline .pre-commit-config.yaml:34]: Add `assets|`

**Review by alexgibson — CHANGES_REQUESTED** [review]
@pmac I didn't pull this locally just yet, but if you want to update dependencies I'll take it for a spin?

File: .pre-commit-config.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
+repos:
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.7
+    hooks:
+      - id: flake8
+        exclude: >
+          (?x)^(
+            docs|
```
Comment (alexgibson) [inline .pre-commit-config.yaml:8]: add `assets|`?

**Comment by pmac** [comment]
Thanks for taking a look and looking into the deps. I'll get it updated.

**Comment by pmac** [comment]
Updated and ready for another try. Working well for me.

**Comment by pmac** [comment]
I removed all references to `static*` because we no longer use those folders, right? Or do we still use static for something?

**Comment by pmac** [comment]
Ah! No, we still do use `static` but not the `static_*` ones. I'll fix.

**Review by alexgibson — APPROVED** [review]
This worked well for me, r+ 💯

**Pull request merged**